dali/operators/reader/loader/video_loader.h

```diff
@@ -295,6 +297,8 @@ class VideoLoader : public Loader<GPUBackend, SequenceWrapper> {
 
   volatile bool stop_;
   std::vector<file_meta> file_info_;
+
+  constexpr static size_t MAX_OPEN_FILE = 200;
```

Comment:
This is an educated guess limit, but maybe it should be just 1/5 of /proc/sys/fs/file-max or have some global counter. What if user opens files in the same process from python? Or spawns more than one VideoReader?
Any suggestion?

Reply:
Maybe we can have it as an argument to the reader so that the user has control of it. Also, I'd help with testing this

Reply:
Maybe `recently_opened_` and `open_files_` should be global to all VideoLoaders - so the limit is global as well. If we have it per operator then I see someone creating multiple VideoReader with the limits that sum up above the OS limit and that leads to a crash.

Reply:
On the other hand we need to have some locking and this can influence the overall perf.